bib/go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/osbuild/bootc-image-builder/bib
 go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.49.18
+	github.com/aws/aws-sdk-go v1.50.0
 	github.com/google/uuid v1.5.0
-	github.com/osbuild/images v0.29.0
+	github.com/osbuild/images v0.32.1-0.20240124173944-a8de56fabb8c
```

Comment:
Note this effectively rolls in https://github.com/osbuild/bootc-image-builder/pull/151 - and it seems to be hitting the same linter failure.  I don't understand it.

Reply:
Yeah, I saw this in a different PR of mine as well :/